scripts/vyatta-clear-squid

```diff
@@ -1,10 +1,10 @@
 #!/bin/sh
 
-CACHE="/var/spool/squid3"
-PID=`cat /var/run/squid3.pid 2>/dev/null`
+CACHE="/var/spool/squid"
+PID=`cat /var/run/squid.pid 2>/dev/null`
 
 if test -n "$PID"; then 
-   /etc/init.d/squid3 stop
+   /etc/init.d/squid stop
    if cli-shell-api existsActive service webproxy url-filtering \
                        squidguard vyattaguard https-filter; then
      echo Stopping VyattaGuard https filter...
```

Comment:
If we are at it, let's also get rid of the mention of VyattaGuard. :)

Reply:
Done